### ISSUE TITLE
Chat / Infobox snapping

### ISSUE THREAD

**Issue by madpew** [srmeier/KnightOnline#95]
twostars pointed out (in my Chat/Info PR) that feature exists and should be implemented as well. (or there's no reason not to do so)

When you drag the infobox within about 10px or so it snaps into place (along the chatwindow) and they both move as a single unit. Moving the infobox (alone) unlocks it, though that requires you to drag it quickly before it'll register you actually want to move it and unlock them.

I'd suggest to simplify this to snapping when withing a range (of 10 px); if they are snapped, moving the chat moves both, moving the infobox unsnaps (in any case regardless of distance and speed).

If noone else want's to tackle this I will put it on my list.

**Comment by twostars** [comment]
Yeah, there's no reason to handle the "dragging quickly" part. It doesn't make a lot of sense. As you've said, best to just do without.
